admin/commands/state_synchronization/execute_script.go

```diff
@@ -0,0 +1,99 @@
+package state_synchronization
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"strconv"
+
+	"github.com/onflow/flow-go/admin"
+	"github.com/onflow/flow-go/admin/commands"
+	"github.com/onflow/flow-go/module/execution"
+)
+
+var _ commands.AdminCommand = (*ReadExecutionDataCommand)(nil)
+
+type scriptData struct {
+	height    uint64
+	script    []byte
+	arguments [][]byte
+}
+
+type ExecuteScriptCommand struct {
```

Comment:
I'm fine with removing this command, but before we have #4781 I need it to test.

Reply:
I think we can keep this command even after #4781 is merged